redfish/eventservice.go

```diff
@@ -361,6 +370,28 @@ func (eventservice *EventService) SubmitTestEvent(message string) error {
 	return err
 }
 
+// SendTestEvent simulates an event using the event service.
+// data specified in the action parameters. This message should then be sent to
+// any appropriate ListenerDestination targets.
+// This action is impended differently by OEM vendors.
+func (eventservice *EventService) SendTestEvent(messageID, oem string) error {
+	var err error
+
+	switch oem {
+	case Dell:
```

Comment:
This is the area I'm concerned with for a couple reasons.

First, the OEM imports should eventually become their own go modules so vendors could potentially supply their own implementations. If/when that happens, that would cause an import loop by those modules declaring a dependency on the main `gofish` module, then the `gofish` module having this dependency of having this awareness of specific OEM implementations.

The second issue is that if these vendors are not following the standard and require their own implementation, then the call should be made using the OEM implementations, not the standard instances. So something like

```go
eventService, _ := serviceRoot.EventService()
hpEventService, _ := hp.FromEventService(&eventService)
hpEventService.SubmitTestEvent("test message")
```

Reply:
removed as asked